proposals/Level-1.md

```diff
@@ -93,19 +92,19 @@ Each exception type has a `type signature`. The type signature defines the list
 of values associated with the exception.
 
 Within the module, exception types are identified by an index into the
-[exception index space](#exception-index-space). This index is referred to as
-the `exception tag`. The `tagged exception type` is the corresponding
+[exception index space](#exception-index-space). This static index refers to the
+corresponding runtime index that uniquely identifies the exception type, and is
+called the `exception tag`. The `tagged exception type` is the corresponding
 exception type refered to by the exception tag.
 
 Exception types can be imported and exported by adding the appropriate entries
 to the import and export sections of the module. All imported/exported exception
 types must be named to reconcile exception tags between modules.
 
-Exception tags are used by:
+Exception indices are used by:
 
-1. The `except` instruction which creates a WebAssembly instance of the
-   corresponding tagged exception type, and pushes a reference to it onto the
-   stack.
+1. The `throw` instruction which creates a WebAssembly instance of the
```

Comment:
Suggestion: I would say it creates an exception value with the corresponding tag.

Reply:
Ok. Reworking the code to match.